src/site/content/en/blog/mediastreamtrack-insertable-media-processing/index.md

```diff
@@ -107,11 +108,10 @@ components:
   [`AudioFrame`](https://w3c.github.io/webcodecs/#audioframe-interface)) objects. You can think of this as a
   track sink that is capable of exposing the unencoded frames from the track as a `ReadableStream`.
   It also exposes a control channel for signals going in the opposite direction.
-- The `MediaStreamTrackGenerator`, which consumes a stream of media frames and exposes a `MediaStreamTrack`
-  interface, so that it can be used in any sink a `MediaStreamTrack` interface.
-  The `MediaStreamTrackGenerator` can be provided to any sink, just like a track from `getUserMedia()`.
-  It takes
-  media frames as input. In addition, it provides access to control signals that are generated by the sink.
+- The `MediaStreamTrackGenerator`, which consumes a stream of media frames and exposes a
+   `MediaStreamTrack` interface. The `MediaStreamTrackGenerator` can be provided to any sink,
```

Comment:
nitpick: This doesn't seem like a complete sentence.

Reply:
Ah, my apologies, that's intentional.

Reply:
Boy, this got butchered badly. Hope it's finally fixed.

Reply:
Yeah, I also just noticed (and fixed it again). Caught the identical beginning of the two sentences, though, so good that we both had a double eye on it.